package.json

```diff
@@ -38,6 +38,7 @@
     "@ethereumjs/common": "^2.3.1",
     "@ethereumjs/tx": "^3.2.1",
     "@metamask/contract-metadata": "^1.33.0",
+    "@ledgerhq/metamask-keyring": "https://github.com/LedgerHQ/metamask-keyring",
```

Comment:
We'll update this in a day or two (once it's published on our side.) Until then this PR is a WIP